### PR TITLE
PhpdocAnnotationWithoutDotFixer - Handle trailing whitespaces

### PR DESCRIPTION
**Pull request by kubawerlos**
Solves #3857 - I guess @SpacePossum will have some more fancy test cases.

### CODE REVIEW

**Comment by SpacePossum** [comment]
this looks awesome to me, thanks so much :)
hope that we get a second review on this

**Comment by SpacePossum** [comment]
(on a side note, if this fixer is about changing text to a sentence without a stop maybe we should unify what we consider a stop (so not only dots), like done [here](https://github.com/FriendsOfPHP/PHP-CS-Fixer/blob/v2.2.20/src/Fixer/Phpdoc/PhpdocSummaryFixer.php#L123))

File: tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php [hunk 1]
```diff
@@ -137,6 +137,16 @@ public function provideFixCases()
...
+     */',
+            ],
+            [
+                '<?php
+    /**
+     * @throws \Exception having whitespaces after dot, yet I am fixed
+     */',
+                '<?php
+    /**
+     * @throws \Exception having whitespaces after dot, yet I am fixed.   '.'
```
Comment (ntzm) [inline tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php]: Please add test case for trailing tab character too, as we are using `\h`

Comment (keradus) [reply tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php]: ping @kubawerlos

Comment (kubawerlos) [reply tests/Fixer/Phpdoc/PhpdocAnnotationWithoutDotFixerTest.php]: Sorry, I've missed that. Done.

**Comment by keradus** [comment]
Thank you @kubawerlos.

**Pull request merged**